mitodl/mitxpro
# style: add favicon and fix certificate partner logo design

### What are the relevant tickets?
https://github.com/mitodl/hq/issues/3409

### Description (What does it do?)
- Adds the favicon
- Fixes design for certificates when there is a partner logo

### Screenshots (if appropriate):



**With Partner Logo (Desktop)**
<img width="1791" alt="image" src="https://github.com/mitodl/mitxpro/assets/34372316/55b1a89c-b8ab-4752-b000-62d8eb3a22a2">


**With Partner Logo (Mobile)**
<img width="428" alt="image" src="https://github.com/mitodl/mitxpro/assets/34372316/44a1712c-12f4-437e-92db-5803d3931e8d">


---
**Without Partner Logo (Desktop)**

<img width="1789" alt="image" src="https://github.com/mitodl/mitxpro/assets/34372316/04c2342e-0a99-4fb7-a0b7-78bf709c637c">


**Without Partner Logo (Mobile)**
<img width="433" alt="image" src="https://github.com/mitodl/mitxpro/assets/34372316/cc77de2d-d428-43ed-b0f3-ffe1e902bcc4">

---

**Browser Tabs (Favicon)**
<img width="1039" alt="image" src="https://github.com/mitodl/mitxpro/assets/34372316/258c3c30-50ca-442a-9414-9375ae304d7d">


### How can this be tested?
- Run the application
- Check the favicon icon is correct
- Create two certificates (With and without Partner Logo)
- Check that the design matches [this Figma Design](https://www.figma.com/file/a8CIg9kGp9jEmkA0BmOFOv/MIT-xPRO-Logo-Update?node-id=145%3A34&mode=dev)
